### PR TITLE
Review for DM-9125: Make SGS: plotsScript.py: makeRachelPlots work!

### PR DESCRIPTION
**Pull request by jdswinbank**
Now that we are tarcking the extendedness classification, we expect the
HSC catalogs to hold 28 rather than 23 columns. This commit modifies
fromDbToTruth accordingly.

### CODE REVIEW

**Comment by jdswinbank** [comment]
NB this PR is based upon `tickets/DM-9002`, not upon master. This branch will need to be rebased after `tickets/DM-9002` has been merged.

**Comment by jdswinbank** [comment]
Typo in commit message (“tarcking”).

File: fromDbToTruth.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 import fsButler.utils as fsUtils
 
-depth = 'udeep'
+depth = 'udeepwide'
```
Comment (jdswinbank) [inline fromDbToTruth.py]: This change doesn't seem to be related to the number of columns in the catalogue. What happened?

File: fromDbToTruth.py [hunk 2]
```diff
@@ -36,8 +36,8 @@
 idCol = np.zeros((fileLen,), dtype=long)
 raCol = np.zeros((fileLen,))
 declCol = np.zeros((fileLen,))
-photArr = np.zeros((fileLen, 20))
-# photArr = np.zeros((fileLen, 25))
+# photArr = np.zeros((fileLen, 20))
```
Comment (jdswinbank) [inline fromDbToTruth.py]: (Here and below) please don't leave commented out code in the script — if it's not necessary, remove it.

File: fromDbToTruth.py [hunk 3]
```diff
@@ -36,8 +36,8 @@
...
+photArr = np.zeros((fileLen, 25))
```
Comment (jdswinbank) [inline fromDbToTruth.py]: Per my earlier review, I'm uncomfortable with the “magic numbers” here. Why 25? 3? 23? 28? etc. I guess these related to the particular properties of your database dump. It would be nice to make them into named variables and describe what each one actually is.

File: fromDbToTruth.py (hunk 1 above)
Comment (AstroVPK) [inline fromDbToTruth.py]: The old catalogs that I had - udeep and deep - were missing the extendedness classification columns. The new catalog that I got from Jim is at (effectively) wide depth and is mocked up using udeep data - hence the name and the change in the catalog being read in.

File: fromDbToTruth.py (hunk 3 above)
Comment (AstroVPK) [inline fromDbToTruth.py]: Taken care of in earlier review (for DM-9002).
